comm/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "comm" # Edit this
+version = "0.1.0"
+authors = ["shisuimadara adyityasingh@gmail.com"] # Edit this
```

Comment:
The format we used so far was `name <email>`

Reply:
Ok, got it, updated.